doc/nrf/includes/boardname_tables/sample_boardnames.txt

```diff
@@ -1,7 +1,7 @@
 .. _sampleboardnames:
 
-Set used by BLE (Central BAS, Central DFU SMP, Central HIDS, Central UART, EnOcean, Mesh Light, Mesh Light Switch, Peripheral HIDS Keyboard,  Peripheral HIDS mouse,
-Peripheral UART, NUS Shell Transport, Throughput) and NFC samples
+Set used by BLE (Central BAS, Central DFU SMP, Central HIDS, Central UART, EnOcean, Mesh Light, Mesh Light Switch, Peripheral HIDS Keyboard, Peripheral HIDS mouse,
```

Comment:
I think @ru-fu is on vacation. @umapraseeda, could you comment?

Reply:
@joerchan @kapi-no The parenthesis contains a full list of Bluetooth LE samples that are supported. This is going to be updated soon according to NCSDK-4950 in an upcoming PR. I agree that instead of listing all the samples, we can keep it generic as suggested. I shall make this change in the future PR.